src/main/java/ti4/buttons/UnfiledButtonHandlers.java

```diff
@@ -2853,18 +2854,21 @@ public static void increaseStrategyCC(ButtonInteractionEvent event, Player playe
 
     @ButtonHandler("proceed_to_strategy")
     public static void proceedToStrategy(ButtonInteractionEvent event, Game game) {
-        if (game.isOmegaPhaseMode() && PriorityTrackHelper.GetPriorityTrack(game).stream().anyMatch(Objects::isNull)) {
-            MessageHelper.sendMessageToChannel(event.getMessageChannel(), "Please fill the priority track before starting the Strategy Phase.");
-            PriorityTrackHelper.PrintPriorityTrack(game);
-            return;
+        String readiedCardsString = "All planets have been readied at the end of the agenda phase.";
+        if (game.isOmegaPhaseMode()) {
+            readiedCardsString = "All cards have been readied at the end of the omega phase.";
+            if (PriorityTrackHelper.GetPriorityTrack(game).stream().anyMatch(Objects::isNull)) {
+                MessageHelper.sendMessageToChannel(event.getMessageChannel(), "Please fill the priority track before starting the Strategy Phase.");
+                PriorityTrackHelper.PrintPriorityTrack(game);
+                return;
+            }
```

Comment:
Minor wording improvement for Omega Phase, where players  ready **all** cards AFTER the agendas, and only planets before the agendas.